### ISSUE TITLE
GUI refresh: share visual language used by other IPFS apps

### ISSUE THREAD

**Issue by lidel** [ipfs-shipyard/ipfs-companion#273]
There was a talk some time ago about using the same UI style as in [ipfs/station](https://github.com/ipfs/station).
Before we start work toward that goal, lets do a quick poll: 

Is this still what we want? (vote: 👍 👎)

**Comment by diasdavid** [comment]
Having one GUI that would enable us to reuse it across Extension, WebUI and Station would be fun. However, now I'm not sure how much viable it would be, probably we would end up with custom views anyway.

That being said, improving the UX of WebUI, Station and IPFS-Companion should be one of the priorities as these applications will be the face of IPFS for many developers.

@lidel if you would like, we can probably bring more people to help you with design and UX :)

**Comment by lidel** [comment]
Yup, views will be different, but what I had in mind was to share "style" /  "visual language", so that Browser Extension, WebUI and Station each "feels like a part of the same family".  

@diasdavid  Hm.. are there plans to refresh UX of WebUI any time soon? 
My thinking is to wait for WebUI refresh, and then adopt its new style:
It seems to be a good first step if we want to make UX polish: WebUI has more views, so it is a better sample for designing a new visual language. 
When done, CSS/assets could be then extracted to a separate artifact and shared between apps.

**Comment by diasdavid** [comment]
> @diasdavid Hm.. are there plans to refresh UX of WebUI any time soon?

Yes there are! :D News incoming soon (later this week)

**Comment by cvan** [comment]
Forgive my ignorance: What are the latest proposed visual changes here? I've seen some mocks in issues and the PRs. Are there specific relevant, recent issues to cross-reference here?

**Comment by lidel** [comment]
@cvan it is an ongoing effort :) Right now [Style Guide and UI Kit are being fleshed out](https://github.com/ipfs-shipyard/pm-ipfs-gui/issues/7), some initial [wireframing work is done for IPFS Desktop app](https://github.com/ipfs-shipyard/pm-ipfs-gui/issues/12) (as it has more UI than this extension) etc. 

I would not expect any revolutionary changes in browser extension until that work is done, and even then it should be mostly a visual polish :sparkles: 

Some references:

- GUI-related work across all things IPFS in [pm-ipfs-gui](https://github.com/ipfs-shipyard/pm-ipfs-gui)  (repo for high level planning)
    - IPFS Brand Book draft: https://github.com/ipfs-shipyard/ipfs-ui-style-guide/issues/3
    - Style Guide/ UI Kit (latest draft)  https://github.com/ipfs-shipyard/pm-ipfs-gui/issues/7#issuecomment-364497449: 
  - If you have questions, ideas or need access to some Google Docs or [Zeplin project](https://app.zeplin.io/project/5a32d45d1a17248135241058), let us know by creating issue in  [pm-ipfs-gui/issues](https://github.com/ipfs-shipyard/pm-ipfs-gui/issues)

**Comment by jessicaschilling** [comment]
Closing in favor of https://github.com/ipfs/ipfs-gui/issues/29 - thanks!
